framework/docker/cosmos/chain.go

```diff
@@ -543,3 +547,18 @@ func (c *Chain) getGenesisFileBz(ctx context.Context, defaultGenesisAmount sdk.C
 
 	return nil, fmt.Errorf("genesis file must be specified if no validator nodes are present")
 }
+
+func (c *Chain) pruneOrphanedVolumes(ctx context.Context) error {
+	report, err := c.Config.DockerClient.VolumesPrune(ctx, filters.Args{})
```

Comment:
the blank filter args means this could remove volumes that are not intended for removal, it could be a problem when running tests in parallel, and also it would remove volumes not created by tastora

Reply:
True